Buildings/Experimental/NatVentControl/Validation.mo

```diff
@@ -0,0 +1,215 @@
+within Buildings.Experimental.NatVentControl;
+package Validation "Validation model for Nat Vent control"
+  model NatVentRoo
```

Comment:
the package `Validation` should not include the model `NatVentRoo`. Don't save contents of package in one file.